sensor_msgs/Image base_image
detect/Candidates[] candidates_list
sensor_msgs/Image depth_image
---
---